core/src/core-api/featuretoggles.js

```diff
@@ -54,7 +46,37 @@ class LuigiFeatureToggles {
    * @example Luigi.featureToggles().getActiveFeatureToggleList();
    */
   getActiveFeatureToggleList() {
+    return [...get(this.featureToggleList)].filter(ft => !ft.startsWith('!'));
```

Comment:
I got your point. However, we need to have this feature to prevent activating a feature toggle which was disabled before. 

The steps:
1. Navigate to `http://localhost:4200/projects/pr1/settings?ft=berlin,munich,!paris`

2 Running `Luigi.featureToggles().getActiveFeatureToggleList()` should return :
['berlin', 'munich']

3 Run `Luigi.featureToggles().setFeatureToggle('paris')` , and `paris` toggle should not be activated.

The way to prevent `paris` be activated is keeping the `disabled` feature toggle(`!paris`) in the list. And prevent it be set.

Reply:
After synced with Philipp, I adjusted the logic. As below
1. `...?ft=!paris` is valid and will be kept in the FT list for preventing `paris` to be active in the future. 
2. Luigi.featureToggles().setFeatureToggle('!paris') will be ignored. Because it starts with 
`!` which is not a valid call.